pandas/core/generic.py

```diff
@@ -5076,6 +5076,9 @@ def pipe(self, func, *args, **kwargs):
         -----
         `agg` is an alias for `aggregate`. Use the alias.
 
+        Pandas operations generally exclude NaNs. For example, ``agg(np.nanmedian)``,
+        ``agg(np.median)``, and ``agg('median')`` will return the same result.
```

Comment:
Thanks @gurukiran07, but I don't find this comment very clear. I don't think we want to talk about what pandas operations generally do in this docstring. I guess you mean that `aggregate` ignores `NaN` values, even if the function argument does not. Just say that, with a code example if you want to be clearer. But I don't expect readers of this docstring to know about `numpy.nanmedian` and `numpy.median` to use them as an example, after a vague sentence about pandas operations in general.

Can you rephrase please?